README.markdown

```diff
@@ -101,27 +101,42 @@ Thirdly, fork away on github.
 ### Setup alonetone locally on macOS
 
 
-1) `clone`
+- Clone this repo
+`git clone REPO_GIT`
 
-2) `brew install libsndfile lame` (required for id3 tags and waveforms)
+- `brew install libsndfile lame` (required for id3 tags and waveforms)
 
-3) `bundle `
+- Install gems
+`bundle install`
 
-4) Create needed config, database, and load db/seeds:
+- To create needed config, database, and load db/seeds*:
+`rake setup`
+- Create and seed database
+`rake db:setup`
 
-      rake setup
+- `rails s`
 
-5) `rails s`
 
+*Note: alonetone uses 3 config files that are created by 'rake setup
 
-Note: alonetone uses 3 config files that are created by 'rake setup'
+> alonetone.yml (contains the application "secret" and app-specific settings)
+> database.yml
+> newrelic.yml (for performance tracking)
 
```

Comment:
Removing the indentation means that we lost the newlines, but I'll merge this PR once it's fixed!